docs/processes/cldr-icu.md

```diff
@@ -415,62 +424,62 @@ appropriate, and repeating from step 4 or 8 as appropriate.
 ## 12 Transfer the ICU4C data to ICU4J
 
 12a. You need to reconfigure ICU4C to include the unicore data.
-```
+```sh
 cd $ICU4C_DIR/source
 ICU_DATA_BUILDTOOL_OPTS=--include_uni_core_data ./runConfigureICU <platform>
 ```
 
 12b. Rebuild the data with the new config setting, then create the ICU4J data jar.
-```
+```sh
 cd $ICU4C_DIR/source/data
 make clean
 make -j -l2.5
 make icu4j-data-install
 ```
 
 12c. Create the  test data jar
-```
+```sh
 cd $ICU4C_DIR/source/test/testdata
 make icu4j-data-install
 ```
 
 12d. Update the extracted {main, test} data files in the Maven build
-```
+```sh
 cd $ICU4J_ROOT
 ./extract-data-files.sh
 ```
 
 ## 13 Rebuild ICU4J with new data, run tests
 
 13a. Run the tests using the maven build
-```
+```sh
 cd $ICU4J_ROOT
 mvn clean
 mvn install 2>&1 | tee $NOTES/icu4j-newData-mvnCheck.txt
 ```
 
 It is possible to re-run a specific test class or method if necessary when fixing bugs.
 
-For example (using artifactId, full class name, test all methods):
-```
+For example (using `artifactId`, full class name, test all methods):
+```sh
 mvn install -pl :core -Dtest=com.ibm.icu.dev.test.util.LocaleBuilderTest
 ```
 or (example of using module path, class name, one method):
-```
+```sh
 mvn install -pl main/common_tests -Dtest=MeasureUnitTest#TestGreek
 ```
 
-13b. Optionally run the tests in exhautive mode
+13b. Optionally run the tests in exhaustive mode
 
 Optionally run before committing changes, or run to diagnose failures from
-running exhastive CI tests in the PR using `/azp run CI-Exhaustive`: 
-```
+running exhaustive CI tests in the PR using `/azp run CI-Exhaustive`:
```

Comment:
`/azp run CI-Exhaustive` is no longer valid, right?

Reply:
good catch... --> https://unicode-org.github.io/icu/userguide/dev/ci.html#exhaustive-tests

could be done in a follow-up PR

Reply:
Oups... sorry. Already did it in this one.

I think it is a left-over from Azure.
The command below the description is correct.

I updated this to what I think was the intention.